RingPublishingTracking/src/main/java/com/ringpublishing/tracking/data/audio/AudioOutput.kt

```diff
@@ -0,0 +1,23 @@
+package com.ringpublishing.tracking.data.audio
+
+/**
+ * Audio output type
+ * @param [text] - type param text
+ */
+enum class AudioOutput(val text: String) {
```

Comment:
generalnie przy player'ze mieliśmy
``` kotlin
        const val MOBILE = "mobile"
        const val CHROMECAST = "chromecast"
        const val CAR = "car"
        const val BLUETOOTH = "bluetooth"
        const val EXTERNAL = "external"
```
warto byłoby tu ustalić spójnie

Reply:
spoko, to pójdę w tę stronę

Reply:
+1